Dockerfile

```diff
@@ -4,9 +4,9 @@ WORKDIR /app
 
 COPY . /app
 
+RUN apt-get update && apt-get install -y libpq-dev build-essential
 RUN pip install -r requirements.txt
-
-COPY . .
+RUN pip install psycopg2-binary
```

Comment:
If you're using SQLAlchemy for ORM-based interaction with PostgreSQL, you don't need psycopg2 or psycopg2-binary. 
In the final project description we have: _PostgreSQL + Optional ORM (SQLAlchemy, etc)_